### DISCUSSION TITLE
How do you connect to a fire tablet without USB?

### DISCUSSION THREAD

**Discussion by randompersononinternet69** [yume-chan/ya-webadb#445] [Q&A]
I have been trying to connect using the IP address, but nothing works when using it.

**Comment by parkerlreed** [comment]
You have to use the websocket intermediate server as mentioned in the documentation 

![Screenshot_20221020_002948_Termux](https://user-images.githubusercontent.com/841440/196863388-236a268c-a6e6-415c-abfc-a97eebe070d3.png)

You can do this on either the client or host device just change the IPs as needed
